data/plugins/linewrapping.lua

```diff
@@ -366,7 +375,10 @@ function DocView:new(doc)
   old_new(self, doc)
   if not open_files[doc] then open_files[doc] = {} end
   table.insert(open_files[doc], self)
-  if config.plugins.linewrapping.enable_by_default then
+  if config.plugins.linewrapping.enable_by_default
+      and (matches_any(self.doc.abs_filename or "", config.plugins.linewrapping.files)
+        or matches_any(self.doc.filename     or "", config.plugins.linewrapping.files))
```

Comment:
`matches_any` is not defined here.

Reply:
Ok, I've added it (again, it is coming from the spellcheck plugin).